text/0000-plugins-1.1.md

```diff
@@ -0,0 +1,486 @@
+- Feature Name: `rustc_macros`
+- Start Date: 2016-07-14
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
```

Comment:
Should this copy of the RFC (with "plugins" in the file name) still be part of the diff?